spark/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+  - Jeffwan
+  - holdenk
```

Comment:
Alphabetical order here?

Reply:
Right, this is against the order. I will fix it